### DISCUSSION TITLE
Listen Message from TcpConnector

### DISCUSSION THREAD

**Discussion by rmanalu15** [reactphp/reactphp#554] [Q&A]
please help.
how to listen message when i use TcpConnector? thank you

**Comment by SimonFrings** [comment]
Hey, @rmanalu15,

Not sure why you're using the TcpConnector in particular, maybe start by using the Connector and follow the example provided inside the project: https://github.com/reactphp/socket/blob/1.x/examples/11-http-client.php
This is also rather similar to the quickstart example inside the documentation.

You can listen for different events on a connection as described here: https://github.com/reactphp/socket?tab=readme-ov-file#connectioninterface

I hope this helps 👍️
